.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
```

Comment:
- [ ] I think, as per the project requierements, It is important to have a test.md file in the project that is mentioned in `.gitignore` so that will not appear on github :thinking: